english/TheSpider/do-homework/study.md

```diff
@@ -0,0 +1,7 @@
+WHAT ARE YOU DOING? 
+
+THERE IS CRIME EVERYWHERE
+
+PUT DOWN THAT BOOK AND TRY AGAIN
+
+[Try again](Spiderman.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.